tracks/mixins.py

```diff
@@ -51,7 +51,7 @@ def get_token(self):
         auth_header = self.request.META.get('HTTP_AUTHORIZATION', '')
         splitted_auth_header = auth_header.split(' ')
 
-        if len(splitted_auth_header):
+        if any(splitted_auth_header):
```

Comment:
Thanks for the heads up. This was, to fix when there was no auth header: `len(''.split(' ')) == 1`